client/src/pages/ProfilePage.jsx

```diff
@@ -0,0 +1,447 @@
+              formikHelpers.setSubmitting(false);
+            });
+        }}
+      >
+        {({ touched, errors, isSubmitting }) => (
+          <Form className="box">
+            <h1 className="title">Change Email</h1>
+            <div className="field">
+              <label htmlFor="newEmail" className="label">
+                New email
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validateEmail}
+                  name="newEmail"
+                  type="email"
+                  id="newEmail"
+                  placeholder="e.g. bobsmith@gmail.com"
+                  className={cn('input', {
+                    'is-danger': touched.newEmail && errors.newEmail,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-envelope"></i>
+                </span>
+
+                {touched.newEmail && errors.newEmail && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.newEmail && errors.newEmail && (
+                <p className="help is-danger">{errors.newEmail}</p>
+              )}
+            </div>
+            <div className="field">
+              <label htmlFor="password" className="label">
+                Password
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validatePassword}
+                  name="password"
+                  type="password"
+                  id="password"
+                  placeholder="*******"
+                  className={cn('input', {
+                    'is-danger': touched.password && errors.password,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-lock"></i>
+                </span>
+
+                {touched.password && errors.password && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.password && errors.password ? (
+                <p className="help is-danger">{errors.password}</p>
+              ) : (
+                <p className="help">At least 6 characters</p>
+              )}
+            </div>
+            <div className="field">
+              <button
+                type="submit"
+                className={cn('button is-success has-text-weight-bold', {
+                  'is-loading': isSubmitting,
+                })}
+                disabled={isSubmitting || errors.email || errors.password}
```

Comment:
The field name 'email' used in the disabled condition should be 'newEmail' to match the form field name.